src/pages/about-us.tsx

```diff
@@ -38,20 +38,23 @@ export const AboutUsPageQuery = graphql`
       }
     }
 
-    allTeamJson {
+    allContentfulTeamMember {
       nodes {
         id
         name
         image {
-          childImageSharp {
-            gatsbyImageData(
-              width: 400
-              aspectRatio: 1
-              layout: CONSTRAINED
-              placeholder: DOMINANT_COLOR
-              formats: [AUTO, WEBP, AVIF]
-            )
-          }
+          gatsbyImageData(
+            width: 400
+            aspectRatio: 1
+            layout: CONSTRAINED
+            placeholder: DOMINANT_COLOR
+            formats: [AUTO, WEBP, AVIF]
+            # people on the photos are not always in the center.
+            # this option from the contentful image api tries to resolve that.
```

Comment:
Actually the comment is not accurate. These are options from gatsby-image and the underlying image process sharp. See [gatsby-plugin-sharp](https://www.gatsbyjs.com/plugins/gatsby-plugin-sharp) and [gatsby-image](https://www.gatsbyjs.com/plugins/gatsby-image/)

While I appreciate that you want to provide a hint to future struggling folks I suggest to remove the comment. People working into this repository should be comfortable querying the gatsby documentation for those keywords 🙏

I also suggest to remove the comment.

```suggestion
```